assets/src/components/developers/EventSubscriptionsPage.tsx

```diff
@@ -160,17 +160,18 @@ class EventSubscriptionsPage extends React.Component<Props, State> {
           onCancel={this.handleWebhookModalCancel}
         />
 
-        {!!personalApiKey && !!accountId && (
-          <Box mb={5}>
-            <Title level={4}>Webhook handler sandbox</Title>
-
-            <CodeSandbox
-              personalApiKey={personalApiKey}
-              accountId={accountId}
-              onSuccess={(data) => this.setState({apiExplorerOutput: data})}
-            />
-          </Box>
-        )}
+        {
+          !!personalApiKey && !!accountId && false
```

Comment:
just prefix the former code with `false` and that'll hide everything, this is kinda weird otherwise